aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -511,6 +518,27 @@ module aptos_framework::voting {
         false
     }
 
+    #[view]
+    public fun get_proposal_metadata<ProposalType: store>(
+        voting_forum_address: address,
+        proposal_id: u64,
+    ): SimpleMap<String, vector<u8>> acquires VotingForum {
```

Comment:
Can you test this out?  I'm not sure how a SimpleMap will be represented on the output of a view function

Reply:
discussed offline, a SimpleMap output in view function will be like
<img width="1003" alt="image" src="https://github.com/aptos-labs/aptos-core/assets/101405096/2ebe0dd1-82f2-4593-8f5a-c946e124c55b">

here's the proposal_votes_breakdown definition:
```
    #[view] /// Table of all votes casted on a proposal
    public fun proposal_votes_breakdown(dao_address: address, proposal_id: u64): SimpleMap<address, ProposalVote> acquires DAO {
        assert!(exists<DAO>(dao_address), ERR_PARLIAMENT_DAO_NOT_EXISTS);
        smart_table::to_simple_map(&smart_table::borrow(&borrow_global<DAO>(dao_address).proposals, proposal_id).casted_votes)
    }
```